src/main/java/editCard/EditCardInputBoundary.java

```diff
@@ -1,14 +1,15 @@
-package editCard;/*
+/*
  * Author: Albert Yan
  * Date: October 31/22
  * This interface allows cards to be edited by higher level classes
  */
 
```

Comment:
see comment in the EditCardController class

Reply:
Done.